### ISSUE TITLE
feat: allow extraVolumes and extraVolumeMounts in helm chart

### ISSUE THREAD

**Issue by hegerdes** [hetznercloud/hcloud-cloud-controller-manager#743]
### TL;DR

The user @simonostendorf  provided the great feature in #652 to read the `HCLOUD_TOKEN` from an file! It is great having the possibility to read the token from a file since leaking files is much less likely then envs.

Unfortunately I have a hard time using this feature when you don't use HashiCorps Secret injector. Normally you would mount the secret as a volume but you cant do this currently since the chart does not support any `extraVolumes` or `extraVolumeMounts`. But this is kind of needed when you use CSI Secret Driver or just want to mount the k8s secret as a file. 

### Expected behavior

Being able to set  `extraVolumes` or `extraVolumeMounts` so I can use my existing k8s secret and mount it as file via secret-key-ref oder to use a kubernetes secret csi driver definition as volume.

Is this something worth supporting? I would happily provide a PR if this is an accepted feature from Hetzner.

**Update:**
PR in #744

**Comment by simonostendorf** [comment]
I think supporting the CSI Secret Driver would be great.

(And cool that someone uses my feature :))

**Comment by apricote** [comment]
Sounds good, happy to accept a pull request for this :)
